services/app-api/forms/mcpar.json

```
@@ -273,7 +273,6 @@
                 }
               ],
               "spreadsheet": "A_Program_Info",
```

json is erroring because of the trailing comma on this line
```suggestion
              "spreadsheet": "A_Program_Info"
```